packages/web-components/src/components/va-accordion/va-accordion.tsx

```diff
@@ -48,15 +49,41 @@ if (Build.isTesting) {
  */
 @Component({
   tag: 'va-accordion',
-  styleUrl: 'va-accordion.css',
+  styleUrl: 'va-accordion.scss',
   shadow: true,
 })
 export class VaAccordion {
   @Element() el!: any;
+  accordionContainer: HTMLDivElement;
   expandCollapseBtn!: HTMLButtonElement;
 
   @State() expanded = false;
 
+  /**
+   * Whether or not the accordion items will have borders
+   */
+  @Prop() bordered?: boolean = false;
+
+  /**
+   * True if only a single item can be opened at once
+   */
+  @Prop() openSingle?: boolean = false;
+
+  /**
+   * Whether or not the component will use USWDS v3 styling.
+   */
+  @Prop() uswds?: boolean = false;
+
+  /**
+   * If `true`, doesn't fire the CustomEvent which can be used for analytics tracking.
+   */
+  @Prop() disableAnalytics?: boolean = false;
+
+  /**
+   * Optional accordion section heading text. Only used in analytics event. Default is null.
+   */
+  @Prop() sectionHeading?: string = null;
+
```

Comment:
The diff makes it seems like all of these props are new to this file. I can understand the `uswds` one but is that also right for the others?

Reply:
This is correct-ish. The only new prop is uswds, but I moved the props to the top of the file instead of having them buried beneath the logic at line 160.

Reply:
Oh yes! They really were buried down there. 😂 Good idea to group them together.